### PR TITLE
Mta 1618 review tab

### PR DESCRIPTION
**Pull request by HagayVider1**
Jira ticket: https://issues.redhat.com/browse/MTA-1618
Replaces https://github.com/windup/windup-documentation/pull/778
Preview: https://deploy-preview-779--windup-documentation.netlify.app/docs/web-console-guide/master/#mta-web-review-tab_user-interface-guide

### CODE REVIEW

File: docs/topics/mta-web-review-tab.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-review-tab_{context}"]
+= Application review tab
+
+You can review the settings of any application or archetype settingd directly from the list.
```
Comment (emarcusRH) [inline docs/topics/mta-web-review-tab.adoc:9]: Minor typo - "archetype settingd"  >> "archetype settings" or "archetype setting"

File: docs/topics/images/mta-web-review-application-01.png
Comment (HagayVider1) [inline docs/topics/images/mta-web-review-application-01.png:1]: Konveyor image not used. To be removed.

File: docs/topics/images/mta-web-review-tab-01.png
Comment (HagayVider1) [inline docs/topics/images/mta-web-review-tab-01.png:1]: Konveyor image not used. To be removed.

File: docs/topics/images/mta-web-review-tab-02.png
Comment (HagayVider1) [inline docs/topics/images/mta-web-review-tab-02.png:1]: Konveyor image not used. To be removed.

File: docs/topics/mta-web-review-tab.adoc (hunk 1 above)
Comment (HagayVider1) [reply docs/topics/mta-web-review-tab.adoc]: Corrected

File: docs/topics/mta-web-review-tab.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+:_content-type: PROCEDURE
+[id="mta-web-review-tab_{context}"]
+= Application review tab
+
+You can review the settings of any application or archetype settings directly from the list.
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Select an application, opening a details pane at the right of the browser window.
```
Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:14]:
```suggestion
. Select an application, opening a `details` pane at the right of the browser window.
```

File: docs/topics/mta-web-review-tab.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
++
+// Get image from MTA UI
+image::mta-web-review-tab.png[Application reviews tab]
+// ![](/Tackle2/AppAssessAnalyze/AnalysisSelect.png)
```
Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:18]: remove comment

File: docs/topics/mta-web-review-tab.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+= Application review tab
+
+You can review the settings of any application or archetype settings directly from the list.
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Select an application, opening a details pane at the right of the browser window.
++
+// Get image from MTA UI
```
Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:16]: remove comment

File: docs/topics/images/mta-web-assessment-tags-02.png
Comment (anarnold97) [inline docs/topics/images/mta-web-assessment-tags-02.png:1]: delete

File: docs/topics/images/mta-web-review-application-01.png
Comment (anarnold97) [inline docs/topics/images/mta-web-review-application-01.png:1]: delete

File: docs/topics/images/mta-web-review-tab-01.png
Comment (anarnold97) [inline docs/topics/images/mta-web-review-tab-01.png:1]: delete

File: docs/topics/images/mta-web-review-tab-02.png
Comment (anarnold97) [inline docs/topics/images/mta-web-review-tab-02.png:1]: delete

File: docs/topics/images/mta-web-review-tab.png
Comment (anarnold97) [inline docs/topics/images/mta-web-review-tab.png:1]: the review items are all `not yet reviewed` making the image pretty worthless
if you are going to include, this has to be populated with an actual example

File: docs/topics/mta-web-review-tab.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-review-tab_{context}"]
+= Application review tab
+
+You can review the settings of any application or archetype settings directly from the list.
```
Comment (nachandr) [inline docs/topics/mta-web-review-tab.adoc:9]: Replace 1 with 2
1)You can review the settings of any application or archetype settings directly from the list.

2)You can view the review details of any application  on the application details page.

**Comment by nachandr** [comment]
Additionally, the following sections are obsolete in MTA 7.0.0

6.1.1. Applying assessments to other applications
6.1.2. Skipping assessment of applications

**Comment by dymurray** [comment]
Agree with @nachandr 's comments

File: docs/topics/mta-web-review-tab.adoc (hunk 5 above)
Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:9]:
```suggestion
You can view the review details of any application on the application details page.
```

Comment (anarnold97) [reply docs/topics/mta-web-review-tab.adoc:9]: @nachandr  - is this what you meant? |
If so i will commit

File: docs/topics/mta-web-review-tab.adoc [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Select an application, opening a `details` pane at the right of the browser window.
++
+// Get image from MTA UI
```
Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:16]:
```suggestion

```

File: docs/topics/mta-web-review-tab.adoc [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+You can review the settings of any application or archetype settings directly from the list.
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Select an application, opening a `details` pane at the right of the browser window.
++
+image::mta-web-review-tab.png[Application reviews tab]
+// ![](/Tackle2/AppAssessAnalyze/AnalysisSelect.png)
```
Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:17]:
```suggestion

```

Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:17]:
```suggestion

```

File: docs/topics/mta-web-review-tab.adoc [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Select an application, opening a `details` pane at the right of the browser window.
++
+image::mta-web-review-tab.png[Application reviews tab]
++
+
+. Click the *Reviews* tab in the pane. The application information appears, showing the application's questions.
```
Comment (jmle) [inline docs/topics/mta-web-review-tab.adoc:19]:
```suggestion
. Click the *Reviews* tab in the pane. The application information appears, showing the items to review.
```

File: docs/topics/mta-web-review-tab.adoc (hunk 5 above)
Comment (HagayVider1) [reply docs/topics/mta-web-review-tab.adoc:9]: Removed line 1 and replace it with line 2.
See https://github.com/windup/windup-documentation/pull/808

**Comment by HagayVider1** [comment]
> Additionally, the following sections should be removed since these features are obsolete in MTA 7.0.0.
> 
> 6.1.1. Applying assessments to other applications 6.1.2. Skipping assessment of applications

Commented out sections in master.adoc

**Pull request closed without merging**